src/button.js

```diff
@@ -58,7 +58,8 @@ function distributeClassPrefix(prefix, classAttr) {
 
 // DOM Event Listeners
 Button.prototype.events = View.prototype.events.extended({
-    click: '_execute'
+    click: '_execute',
+    keyup: '_execute'
```

Comment:
the _executes that matter for this are https://github.com/Livefyre/streamhub-share/pull/18/files#diff-e49934041bf3f89885a6ca2851859385R43 and https://github.com/Livefyre/streamhub-share/pull/18/files#diff-b5a7a652d1b60a32137bda9f9f85643fR94 

I can update the _execute in this project, but it isn't being used by Media Wall and needing aria updates.